### ISSUE TITLE
Cannot publish using `npm run publish`

### ISSUE THREAD

**Issue by smsimone** [raycast/extensions#4954]
### Description

I'm trying to publish my [extension](https://github.com/smsimone/raycast_mixpanel) but when I try to run `npm run publish` I keep getting this result:

```
@smaso ➜ raycast_mixpanel git(main) npm run publish          

> publish
> ray publish

ready -  validate git repository
error -  getting fork
fork extensions repo: fork not ready after 1min
```

**Comment by mathieudutour** [comment]
does it still fails if you try again?

**Comment by smsimone** [comment]
tried again right now but the behaviour is still the same

**Comment by mathieudutour** [comment]
Should be fixed in 1.48.4
